AIMSICD/src/main/res/values-it/translatable_strings.xml

```diff
@@ -0,0 +1,427 @@
+
+    <!-- Section Application -->
+    <string name="help_app_add_get_ocid_api_key">Ottieni una chiave API per usare il database OpenCellID per caricare e scaricare dati…</string>
+    <string name="help_app_about">Icone di stato e Project spiegato.</string>
+    <string name="help_app_download_local_bts">Scarica il database OpenCellID.</string>
+    <string name="help_app_upload_local_bts">Carica il tuo BTS nel database OpenCellID.</string>
+    <string name="help_app_debugging">Colleziona e manda il logs degli errori.</string>
+    <string name="help_app_quit">Esci dall\'applicazione.</string>
+    <string name="waiting_for_location">Aspetta per la posizone…</string>
+    <string name="unable_to_restore_backup_from_previous_database_version">impossibile ripristinare il salvataggio del database dalla versione precedente a causa di cambiamenti strutturali!</string>
+    <string name="unique_bts_data">Unico BTS Dati</string>
+    <string name="bts_measurements">Misure BTS</string>
+    <string name="imported_ocid_data">Dati OpenCellID importati</string>
+    <string name="default_mmc_locations">Posizioni MCC di Default</string>
+    <string name="silent_sms">Silent SMS</string>
+    <string name="eventlog">Log degli eventi</string>
+    <string name="measured_signal_strengths">Forza segnale misurato</string>
+
+    <string name="at_command_response_looking">*** Cercando dispositivi seriali AT…\n</string>
+    <string name="at_command_response_found">Trovato: </string>
+    <string name="at_command_response_setup_complete">*** Configurazione Completata ***\n</string>
+
+    <string name="currently_connected_bts">BTS Attualmente Connesso</string>
+    <string name="well_known_encrypted_bts">BTS noto e cripato</string>
+    <string name="unknown_or_insecure_bts">BTS sconosciuto o non sicuro</string>
+    <string name="new_or_recently_added_bts">BTS nuovo o recentemente aggiunto</string>
+
+    <!-- Section Delete DB -->
+    <string name="delete_database_msg_success">Database cancellato e ricreato successivamente.\nPerfavore riavvia l\'applicazione per cancellare i valori vecchi presenti in memoria.</string>
+
+    <!-- Section OCID Download API key -->
+    <string name="invalid_key_try_later">Chiave OpenCellId non valida, riprova più tardi.</string>
+    <string name="getting_ocid_key">Ottenendo chiave API OpenCellId…</string>
+
+    <!-- Advanced user menu -->
+    <string name="deleted">Cancellato</string>
+    <string name="failed_to_delete">Fallito la rimozione</string>
+    <string name="double_quote_will_cause_db_error">"Stringa non aggiunta\n \" doppia quota causa errori al database "</string>
+    <string name="the_string_was_added_to_db">La stringa è stata aggiunta al Database</string>
+    <string name="failed_to_add_the_string_to_db">Fallito l\'aggiunta della stringa nel Database</string>
+    <string name="n_a">N/A</string>
+
+    <string-array name="pref_ui_icons_entries">
+        <item>@string/ui_icon_flat</item>
```

Comment:
i didn't translate just all strings as an Translator, i also interpretate the context, therefore if you use an translator maybe will give you different words.
initialy i made the fork of AIMSICD for study the code and do some changes, and i took the oppotunity for traslate it.